docs/src/pages/components/snackbars/snackbars-pt.md

```diff
@@ -75,4 +75,33 @@ Para usos mais avançados, você pode tirar vantagem com:
 
 No exemplo a seguir, demonstramos como usar [notistack](https://github.com/iamhosseindhv/notistack). O notistack facilita a exibição de snackbars (para que você não tenha que lidar com o estado de abertura / fechamento delas). Também permite empilhá-las umas sobre as outras (embora isso seja desencorajado pela especificação).
 
-{{"demo": "pages/components/snackbars/IntegrationNotistack.js"}}
\ No newline at end of file
+{{"demo": "pages/components/snackbars/IntegrationNotistack.js"}}
+
+## Accessibility
+
+(WAI-ARIA: https://www.w3.org/TR/wai-aria-1.1/#alert)
+
+- Since alerts are not required to receive focus, content authors should not require users to close a Snackbar if the role is set to `alert` through the SnackbarContent `role` prop.
+- If a Snackbar requires focus to close it, then content authors should use the default `role` of `alertdialog`.
```

Comment:
Please don't edit the translations files, they are keep in sync by a bot.

Reply:
Ok, sure! I don't see the translation files in this PR anymore though; I didn't add them back after you removed them.

Reply:
Agree, it's strange

Reply:
Maybe a squash and rebase will solve it 🤔. Anyway, it's not very important.

Reply:
I removed the translations Accessibility section. They still said the default role was `alertdialog` so I didn't want to risk that confusing anyone.